app/dependencies.php

```diff
@@ -36,8 +36,8 @@
     $container['Database'] = new App\Utilities\Database();
 
     $container['ProductModel'] = DI\factory('App\Factories\ProductModelFactory');
-
     $container['AddProductController'] = DI\factory('App\Factories\AddProductControllerFactory');
+    $container['GetProductsController'] = DI\factory('App\Factories\GetProductsControllerFactory');
 
     $containerBuilder->addDefinitions($container);
 };
```

Comment:
Whitespace needed on the bottom of all files

Reply:
Yeah there is a whitespace at the bottom of the file, it's just not shown on github. there woul be a red error circle thingy if there wasn't